### ISSUE TITLE
About training  epoches  on custom data 

### ISSUE THREAD

**Issue by cch2016** [HRNet/HRNet-Semantic-Segmentation#38]
I am training on my data, training is 7k.  About 40 epoches, my val mIoU is only about 0.37 and some class IoU is 0 . In your paper , training is 2975 and about 484 epoches. I wonder if  need the same epoches or if there's a problem with my data.

**Comment by sunke123** [comment]
Actually, I have no idea what results in the much low performance.

If you data is all-right, you should check the training loss/accuracy curve and val/test loss/accuracy curve to adjust the \#epoch, lr, and other related hyper-parameters.

**Comment by SSJIACV** [comment]
> 
> 
> I am training on my data, training is 7k. About 40 epoches, my val mIoU is only about 0.37 and some class IoU is 0 . In your paper , training is 2975 and about 484 epoches. I wonder if need the same epoches or if there's a problem with my data.

how do you generate the train.lst ?
can you show me the code ?
thank you very muach!!
